### PR TITLE
Added chat and forum features

### PR DESCRIPTION
**Pull request by Clairvoyant1236**
@yilugit 
@doraemon0

### CODE REVIEW

**Comment by yilugit** [comment]
need to fix the build @Clairvoyant1236

**Comment by yilugit** [comment]
👍

File: README_CN.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 [![Build Status](https://travis-ci.org/U-Network/UNetwork.svg?branch=master)](https://travis-ci.org/U-Network/UNetwork)
 
-# UNetwork 
+
+
+# UNetwork (Distributed Networks Architecture)
+>>>>>>> Changed UGCNetwork -> UNetwork
```
Comment (doraemon0) [inline README_CN.md]: Remove this line。

**Comment by doraemon0** [comment]
Please make a new commit to remove unuseful JSON-RPC interfaces in file net/httpjsonrpc/interfaces.go, including following function:
createWallet
openWallet
closeWallet
recoverWallet
getWalletKey

**Comment by doraemon0** [comment]
https://github.com/U-Network/UNetwork/issues/13
This issue could be closed, it is already fixd it in your commit.

**Pull request merged**